product/routes.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: "Routes (beta)"
```

Comment:
```suggestion
title: "Routes (Beta)"
```

Does Mintlify not provide a way to add a badge next to page titles and links in the navigation?